docs/api/post-v1-determineversion.md

```diff
@@ -47,7 +47,9 @@ We recommend trying the API endpoint with our [indexer-api-caller](https://githu
 
 1. Have a local copy of this repostiory.
 2. Navigate to `/osv.dev/tools/indexer-api-caller`
-3. Run the tool using `go run . -lib path/to/library`
+3. Run the tool with the following commands:
+  - For a single library: `go run . -lib path/to/library`
+  - For a directory with multiple libraries: `go run . -dir /path/to/libs/dir`
```

Comment:
nit: multiple libraries **as top level subdirectories**. We can't recurse down them yet.